### ISSUE TITLE
Azure Monitor: Log Analytics suggestion selection broken

### ISSUE THREAD

**Issue by maxilampert** [grafana/grafana#19959]
**What happened**:
Tested for Log Analytics while typing a table name the suggested table name is displayed but after clicking on it the suggestion disappears and the name in the editor is not completed, this error is logged in the DevTools

![image](https://user-images.githubusercontent.com/1974396/67331050-94c05480-f51d-11e9-8ce9-f487fed6371a.png)

**What you expected to happen**:
Suggestion to be copied in the query editor correctly

**How to reproduce it (as minimally and precisely as possible)**:
Create new Log Analytics query 
Type table name, wait for suggestion to be displayed and click on it
**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.4.3
- Data source type & version: Azure Monitor
- OS Grafana is installed on:
- User OS & Browser: Windows & Edge Chrome
- Grafana plugins:
- Others:

**Comment by torkelo** [comment]
Are you using the latest version of Azure Monitor?

**Comment by maxilampert** [comment]
@torkelo yes tested with Grafana 6.4.3 and latest master issue is in both

**Comment by kylebrandt** [comment]
Can replicate this still in current master, 21971a4df8, so still an active issue.

**Comment by joshhunt** [comment]
In Grafana 8.0 we've switched to a different editor component which uses a new way to do suggestions which should have addressed this. I gave this a test again and the suggestions worked as expected

If you're still having issues around this, reopen the issue with some details about the schema of your Log Analytics workspace, or the resource you're trying to query.  Also, checking the behaviour of the editor in the Azure Portal to see how Grafana compares to that would be helpful.
